zerver/lib/actions.py

```diff
@@ -5008,6 +5023,9 @@ def do_create_realm(
             realm.demo_organization_scheduled_deletion_date = (
                 realm.date_created + datetime.timedelta(days=settings.DEMO_ORG_DEADLINE_DAYS)
             )
+
+        # Changing default permission policy for education organisations
+        realm = set_education_realm_permissions(realm)
```

Comment:
Once you stop returning the realm, this assignment should no longer be needed.

Reply:
I forgot that python doesn't require this type of assignment  :(, I think I have been c++ for so long. I'll change this :)